test/e2e/autoscale_test.go

```diff
@@ -283,7 +262,8 @@ func TestAutoscaleUpDownUp(t *testing.T) {
 		clients.KubeClient,
 		deploymentName,
 		isDeploymentScaledUp(),
-		"DeploymentScaledUp")
+		"DeploymentScaledUp",
+		2*time.Minute)
```

Comment:
Same comment with regards to the 2 minute timeout as the previous scaleup block. Lower is better, as long as the tests don't flake with the lower value.

Reply:
Yeah, same deal.  All the requests have succeeded.  Now we're just giving the autoscaler time to respond.  The metrics pipeline has a 60 second window, so 2 minutes is definitely enough to scale up.